src/main/java/se/sundsvall/chat/integration/db/model/MetaDataEntity.java

```diff
@@ -0,0 +1,95 @@
+package se.sundsvall.chat.integration.db.model;
+
+import jakarta.persistence.CollectionTable;
+import jakarta.persistence.Column;
+import jakarta.persistence.ElementCollection;
+import jakarta.persistence.Entity;
+import jakarta.persistence.ForeignKey;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.Table;
+import java.util.List;
+import java.util.Objects;
+import org.hibernate.annotations.UuidGenerator;
+
+@Entity
+@Table(name = "conversation_meta_data")
+public class MetaDataEntity {
+
+	@Id
+	@UuidGenerator
+	@Column(name = "id")
+	private String id;
+
+	@Column(name = "\"key\"")
+	private String key;
+
+	@ElementCollection
+	@CollectionTable(name = "conversation_meta_data_values", joinColumns = @JoinColumn(name = "conversation_meta_data_id"), foreignKey = @ForeignKey(name = "fk_meta_data_values"))
```

Comment:
conversation_meta_data_values
->
conversation_metadata_values"

Update fk-name and join column name accordingly